### ISSUE TITLE
User Assigned Managed Identities: uai vs id slug

### ISSUE THREAD

**Issue by cveld** [Azure/terraform-azurerm-naming#125]
This library has defined `uai` as the slug for user assigned managed identities, whereas the cloud adoption framework abbreviation suggestions list state `id`: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

Are you planning to consolidate the guidance from cloud adoption framework?
Can you extend the module in such way that it is possible for us a user of your module to pick `id` as the slug?

**Comment by RolfMoleman** [comment]
There is actually a separate CAF naming module

**Comment by cveld** [comment]
@RolfMoleman do you have a pointer?

**Comment by mindfulmonk** [comment]
Currently there are 3 conventions it seems:

- uai - used by this repo
- uami - used by CAF repo https://github.com/search?q=repo%3AAzure%2Fterraform-azurerm-caf-enterprise-scale+uami&type=code 
- id - documented in the upstream docs

Naming is hard :) I would vote for uami, def not `id` as it makes no-sense.

**Comment by Nepomuceno** [comment]
You're right that CAF uses `id`. This is a breaking change that will be tracked for the next major version.

**Comment by t-l-k** [comment]
> You're right that CAF uses `id`. This is a breaking change that will be tracked for the next major version.

CAF uses `id` or `uami` ? Confused 😵‍💫

@Nepomuceno What would the breaking change be? Alignment with the CAF? `uami` by the looks of their repo reference...
